.eslintrc

```diff
@@ -22,6 +22,9 @@
         "allowAny": true,
       },
     ],
-    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      { "argsIgnorePattern": "^_" },
```

Comment:
why do we need this?

Reply:
so we can selection postional arguments, for example `keys.map(_, index => ....)`

Reply:
We were doing it in the validation function I think but no longer needs so I reverted the lint config.